helm/nginx-ingress-controller-app/templates/controller-deployment.yaml

```diff
@@ -107,6 +105,9 @@ spec:
         {{- if .Values.controller.disableExternalNameForwarding }}
         - --disable-svc-external-name
         {{- end }}
+        {{- if .Values.controller.watchIngressWithoutClass }}
+        - --watch-ingress-without-class=true
+        {{- end }}
```

Comment:
I would comment this as work around for customers cannot migrate to spec field or annotation quickly

Reply:
Good idea. I'll add a section in the README!

Reply:
Can you check out https://github.com/giantswarm/nginx-ingress-controller-app/pull/218/commits/d0cd5a8dd8e8e0e6f781db5eaa56c1aaff1a85ce